modules/monitoring-resources-reference-for-the-cluster-monitoring-operator.adoc

```diff
@@ -0,0 +1,156 @@
+
+=== openshift-user-workload-monitoring/alertmanager-user-workload
+
+Expose the user-defined Alertmanager web server within the cluster on the following ports:
+
+* Port 9095 provides access to the Alertmanager endpoints. Granting access requires binding a user to the `monitoring-alertmanager-api-reader` role (for read-only operations) or `monitoritoring-alertmanager-api-writer` role in the `openshift-user-workload-monitoring` project.
+* Port 9092 provides access to the Alertmanager endpoints restricted to a given project. Granting access requires binding a user to the `monitoring-rules-edit` cluster role or `monitoring-edit` cluster role in the project.
+* Port 9097 provides access to the `/metrics` endpoint only. This port is for internal use, and no other usage is guaranteed.
+
+=== openshift-monitoring/alertmanager-main
+
+Expose the Alertmanager web server within the cluster on the following ports:
+
+* Port 9094 provides access to all the Alertmanager endpoints. Granting access requires binding a user to the `monitoring-alertmanager-edit` role in the `openshift-monitoring` project.
+* Port 9092 provides access to the Alertmanager endpoints restricted to a given project. Granting access requires binding a user to the `monitoring-rules-edit` cluster role or `monitoring-edit` cluster role in the project.
+* Port 9097 provides access to the `/metrics` endpoint only. This port is for internal use, and no other usage is guaranteed.
+
+=== openshift-monitoring/kube-state-metrics
+
+Expose kube-state-metrics `/metrics` endpoints within the cluster on the following ports:
+
+* Port 8443 provides access to the Kubernetes resource metrics. This port is for internal use, and no other usage is guaranteed.
+* Port 9443 provides access to the internal kube-state-metrics metrics. This port is for internal use, and no other usage is guaranteed.
+
+=== openshift-monitoring/metrics-server
+
+Expose the metrics-server web server on port 443. This port is for internal use, and no other usage is guaranteed.
+
+=== openshift-monitoring/monitoring-plugin
+
+Expose the monitoring plugin service on port 9443. This port is for internal use, and no other usage is guaranteed.
+
+=== openshift-monitoring/node-exporter
+
+Expose the `/metrics` endpoint on port 9100. This port is for internal use, and no other usage is guaranteed.
+
+=== openshift-monitoring/openshift-state-metrics
+
+Expose openshift-state-metrics `/metrics` endpoints within the cluster on the following ports:
+
+* Port 8443 provides access to the OpenShift resource metrics. This port is for internal use, and no other usage is guaranteed.
+* Port 9443 provides access to the internal `openshift-state-metrics` metrics. This port is for internal use, and no other usage is guaranteed.
+
+=== openshift-monitoring/prometheus-adapter
+
+Expose the `prometheus-adapter` web server on port 443. This port is for internal use, and no other usage is guaranteed.
+
+=== openshift-monitoring/prometheus-k8s
```

Comment:
prometheus-k8s service also have 9092 /metrics port, do we need to mention?
```
$ oc -n openshift-monitoring get svc prometheus-k8s
NAME             TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
prometheus-k8s   ClusterIP   172.30.178.48   <none>        9091/TCP,9092/TCP   5h47m

$ oc -n openshift-monitoring get svc prometheus-k8s -oyaml
...
  ports:
  - name: web
    port: 9091
    protocol: TCP
    targetPort: web
  - name: metrics
    port: 9092
    protocol: TCP
    targetPort: metrics
...
```

Reply:
@simonpasquier WDYT?

Reply:
I wouldn't document it since it's purely internal.

Reply:
Added